src/screens/Cart/Cart.js

```diff
@@ -0,0 +1,58 @@
+import React, { Component } from 'react';
+import { Text, View, ScrollView, ActivityIndicator } from 'react-native';
+import CustomButton from 'src/components/CustomButton';
+import Icon from 'react-native-vector-icons/FontAwesome';
+import styles from './styles';
+import { getFromCart } from 'src/api/product';
+import { getFromStorage } from 'src/helpers/asyncStorage';
+
+class Cart extends Component {
+  state = { items: [], loaded: false };
+
+  navigateToList = () => this.props.navigation.navigate('ProductsList');
+
+  componentDidMount() {
+    this.loadCart();
+  };
+
+  loadCart = async () => {
+    try {
+      const token = await getFromStorage('token');
+      const items = await getFromCart(token);
+      this.setState({ loaded: true, items });
+    } catch (e) {
+      console.log(e);
+    }
+  };
+
+  renderItem = (item) => (
+    <View key={item.item_id} style={styles.product}>
+      <View style={styles.productTitleContainer}>
+        <Icon size={17} name={'map'} />
+        <Text style={styles.text}>{item.name}</Text>
+      </View>
+    </View>
+  );
+
+  render() {
+    const { loaded, items } = this.state;
+    return (
+      loaded
+        ? <ScrollView>
```

Comment:
Use ScrollView as much rare as possible. It has some performance issues for large lists. Use FlatList or SectionList instead